docs/pages/experiments/joy/card.tsx

```diff
@@ -0,0 +1,543 @@
+          }}
+        >
+          <Card size="lg" component="li" sx={{ '--Card-radius': '8px' }}>
+            <CardContent>
+              <Typography level="h2" sx={{ fontSize: 'lg', mb: 1, alignSelf: 'flex-start' }}>
+                Bahamas Islands
+              </Typography>
+              <IconButton
+                aria-label="bookmark Bahamas Islands"
+                variant="text"
+                color="neutral"
+                sx={{ position: 'absolute', top: '1rem', right: '1rem' }}
+              >
+                <BookmarkAdd />
+              </IconButton>
+              <Typography level="body2">24 April - 02 May, 2021</Typography>
+              <Typography level="body3">Category/Ocean</Typography>
+              <Typography sx={{ mt: 3 }}>Avarage Temperature</Typography>
+              <Box sx={{ width: 24, height: 2, bgcolor: 'text.primary', mt: 0.5, mb: 1.5 }} />
+              <Typography
+                level="h5"
+                endDecorator={<Sun sx={{ color: '#f3be77' }} />}
+                sx={{ fontWeight: 'md' }}
+              >
+                +18°C
+              </Typography>
+              <AspectRatio min="120px" max="200px" sx={{ my: 2 }}>
+                <img
+                  src="https://images.unsplash.com/photo-1501698335706-90b736210a61?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1674&q=80"
+                  alt=""
+                />
+              </AspectRatio>
+              <Box sx={{ display: 'flex' }}>
+                <div>
+                  <Typography level="body3">Total Price</Typography>
+                  <Typography level="h5" sx={{ fontWeight: 'lg' }}>
+                    $2950
+                  </Typography>
+                </div>
+                <Button
+                  variant="outlined"
+                  color="neutral"
+                  aria-label="Explore Bahamas Islands"
+                  sx={{ ml: 'auto' }}
+                >
+                  Explore
+                </Button>
+              </Box>
+            </CardContent>
+          </Card>
+          <Card
+            component="li"
+            size="lg"
+            sx={{ minHeight: '360px', '&:hover': { boxShadow: 'xl' } }}
+          >
+            <CardCover>
+              <img
+                src="https://images.unsplash.com/photo-1525630558331-067c957817a9?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2250&q=80"
+                alt=""
+              />
+              {gradient}
+            </CardCover>
+            <CardContent sx={{ justifyContent: 'flex-end' }}>
+              <Typography level="h2" sx={{ mb: 1, fontSize: 'lg', color: 'neutral.50' }}>
+                <Link href="#the-beach" underline="none" overlay sx={{ color: 'inherit' }}>
+                  The Beach
+                </Link>
+              </Typography>
+              <Box sx={{ display: 'flex', gap: 2 }}>
```

Comment:
Semantically, the Stack could be clearer but it's more a matter of taste, I assume.